docs.feldera.com/src/pages/markdown-page.md

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
what is this page?

Reply:
nobody knows it's coming from docs.feldera.com repo